x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/snapshot_policies/snapshot_policies.tsx

```diff
@@ -0,0 +1,157 @@
+  const { error, isLoading, data, sendRequest } = useLoadSnapshotPolicies();
+
+  const policies = data.map((name: string) => ({
+    label: name,
+    value: name,
+  }));
+
+  const onComboChange = (options: EuiComboBoxOptionOption[]) => {
+    if (options.length > 0) {
+      onChange(options[0].label);
+    } else {
+      onChange('');
+    }
+  };
+
+  const onCreateOption = (newValue: string) => {
+    onChange(newValue);
+  };
+
+  let calloutContent;
+  if (error) {
+    calloutContent = (
+      <Fragment>
+        <EuiSpacer size="m" />
+        <EuiCallOut
+          data-test-subj="policiesErrorCallout"
+          size="s"
+          iconType="help"
+          color="warning"
+          title={
+            <Fragment>
+              <FormattedMessage
+                id="xpack.indexLifecycleMgmt.editPolicy.deletePhase.noPoliciesLoadedTitle"
+                defaultMessage="Couldn't load existing policies."
+              />
+
+              <EuiButtonIcon
+                size="s"
+                color="warning"
+                onClick={sendRequest}
+                iconType="refresh"
+                aria-label={i18n.translate(
+                  'xpack.indexLifecycleMgmt.editPolicy.deletePhase.reloadPoliciesLabel',
+                  {
+                    defaultMessage: 'Reload policies',
+                  }
+                )}
+              />
+            </Fragment>
+          }
+        >
+          <FormattedMessage
+            id="xpack.indexLifecycleMgmt.editPolicy.deletePhase.noPoliciesLoadedMessage"
+            defaultMessage="You still can type in a policy name."
+          />
+        </EuiCallOut>
+      </Fragment>
+    );
+  } else if (data.length === 0) {
+    calloutContent = (
+      <Fragment>
+        <EuiSpacer size="m" />
+        <EuiCallOut
+          data-test-subj="noPoliciesCallout"
+          size="s"
+          iconType="help"
+          color="warning"
+          title={
+            <FormattedMessage
+              id="xpack.indexLifecycleMgmt.editPolicy.deletePhase.noPoliciesCreatedTitle"
+              defaultMessage="No policies"
+            />
+          }
+        >
+          <FormattedMessage
+            id="xpack.indexLifecycleMgmt.editPolicy.deletePhase.noPoliciesCreatedMessage"
+            defaultMessage="You haven't created any snapshot policies yet. You still can type in a policy name."
+          />
+        </EuiCallOut>
+      </Fragment>
+    );
+  } else if (value && !data.includes(value)) {
+    calloutContent = (
+      <Fragment>
+        <EuiSpacer size="m" />
+        <EuiCallOut
+          data-test-subj="customPolicyCallout"
+          size="s"
+          iconType="help"
+          color="warning"
+          title={
+            <FormattedMessage
+              id="xpack.indexLifecycleMgmt.editPolicy.deletePhase.customPolicyTitle"
+              defaultMessage="Your input doesn't match any existing snapshot policies."
```

Comment:
```suggestion
              defaultMessage="Policy name not found."
```

Reply:
@yuliacech, can you please remove the period after "found"? I meant to remove it in my review.